apps/www/_blog/2023-11-01-supabase-is-now-compatible-with-nextjs-14.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Supabase is now compatible with Next.js 14
+description: The fastest way to build apps with Next.js 14 and Supabase
+author: jonmeyers_io
+categories:
+  - product
+tags:
+  - nextjs
+  - intgrations
+date: '2023-11-01'
+toc_depth: 3
+image: todo
+thumb: todo
+---
+
+As part of [Next.js Conf 2023](https://nextjs.org/conf), the team at Vercel released [Next.js 14](https://nextjs.org/blog/next-14). The huge headling feature was...
+
+<img
```

Comment:
we have a component for this right ?

@fsansalvadore

Reply:
Oh sick! Do we? I tried to use some MDX components I was used to from the docs but couldn't get them to work. Do you have an example @MildTomato or @fsansalvadore?